src/readme.txt

```diff
@@ -1,23 +1,63 @@
-=== WordPress.com Migration ===
-Contributors: wordpress.com, blogvault, akshatc
-Tags: wordpress.com, migration
-Requires at least: 4.0
-Tested up to: 6.6
-Requires PHP: 5.6.0
-Stable tag: 5.65
+=== Move to WordPress.com ===
+Contributors: automattic
+Tags: migrate, migration, backup, restore, transfer, move, copy, wordpress.com, automattic, import, importer, hosting
+Requires at least: 6.3
+Requires PHP: 7.0
+Tested up to: 6.4
+Stable tag: 2.0.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
+A WordPress plugin that helps users to migrate their sites to WordPress.com
+
 == Description ==
```

Comment:
We might want to make this a title (## Description) to match the other titles we have on the page.

Reply:
This matches the format of the existing readme.txt for the Move to WordPress plugin (https://mc.a8c.com/pb/3581f/#plain).

I think it has to be that way to meet the .org [plugin readme standards.](https://wordpress.org/plugins/readme.txt)

Reply:
Aah ok. Sorry to have you do this. We should likely update the other titles to match this formatting to be consistent then.

Reply:
Good point, I didn't think of that when I was looking at it 😝 . I'll fix those shortly.

Reply:
Fixed!